terraform/ec2.tf

```diff
@@ -80,9 +81,23 @@ resource "aws_instance" "snapshot-streaming" {
     destination = "/tmp/snapshot-streaming.service"
   }
 
+   provisioner "file" {
+    content = templatefile("templates/clean_indices.sh", {
+      opensearch-url = var.opensearch-url
+    })
+    destination = "/home/ec2-user/snapshot-streaming/clean_indices.sh"
+  }
+
+  provisioner "file" {
+    source = "templates/restart"
+    destination = "/home/ec2-user/snapshot-streaming/restart"
+  }
+
   provisioner "remote-exec" {
     inline = [
-      "sudo chmod 774 /home/ec2-user/snapshot-streaming/start",
+      "sudo chmod 774 /home/ec2-user/snapshot-streaming/start.sh",
```

Comment:
Why?

Reply:
There were already two files, so you could add the third one in the same pattern

Reply:
Which pattern do you think is better? Should I change it back?

Reply:
Adding the new file I'd keep the existing pattern (and we use it across all projects and nodes)